packages/language-core/src/generators/template.ts

```diff
@@ -1365,6 +1365,11 @@ export function generate(
 					continue;
 				}
 
+				if (vueCompilerOptions.target < 3 && node.tag === 'transition' && prop.name === 'persisted') {
```

Comment:
What will happen the tag is `Transition`?

Reply:
Good point. I thought it's normalized by the compiler.